src/aura/gs_homepageTemplate/gs_homepageTemplate.cmp

```diff
@@ -0,0 +1,28 @@
+<aura:component implements="lightning:homeTemplate"
```

Comment:
All of the LWC components use a naming format without the underscore. For example, `gsAdminSetup`. I think it would be better if this file followed the same naming pattern as the other components.